tests/unit/test_settings_health.py

```
@@ -0,0 +1,169 @@
+# pyright: reportMissingImports=false
+"""Tests for settings health and progress SSE endpoints (_health.py)."""
+
+import time
+from datetime import datetime, timedelta
+from unittest.mock import MagicMock
+
+
+class TestHealthPlugins:
+    def test_filters_stale_entries(self, client, monkeypatch):
```

Ruff (0.15.7)

[warning] Unused method argument: `monkeypatch`

(ARG002)